db/DBHelper.py

```diff
@@ -0,0 +1,150 @@
+import postgresql as pg
+from os import listdir
+from db.date.ItemInfo import ItemInfo
+
+CHECKS_TABLE = "\"Checks\""
+USERS_TABLE = "\"Users\""
+ITEMS_TABLE = "\"Items\""
+CATEGORIES_TABLE = "\"Categories\""
+PATTERNS_TABLE = "\"Patterns\""
+
+USER = "postgres"
+PASS = ""
+HOST = "localhost"
+PORT = "5432"
+DB_NAME = "QrCodesDB"
+
+DB_PARAMS = 'pq://' + USER + ':' + PASS + '@' + HOST + ':' + PORT + '/' + DB_NAME
+
+SCRIPTS_PATH = "scripts/"
+SEQ_SCRIPTS_PATH = SCRIPTS_PATH + "seq/"
+TAB_SCRIPTS_PATH = SCRIPTS_PATH + "tables/"
+
+
+# TODO: methods for statistic
+# TODO: methods for waiting chacks
+
+
+class DBHelper:
+
+    def __init__(self, local=True, user="", password="", host="", port="", db_name=""):
+        try:
+            if local:
+                try:
+                    self.__create_new_db()
+                    self.db = self.__conect_to_local()
+                except pg.exceptions.ClientCannotConnectError:
+                    print("<INFO>: Attempt to create a new database:\n" + DB_PARAMS)
+                    self.db = self.__create_new_db()
```

Comment:
```suggestion
                    self.__create_new_db()
```
функция `__create_new_db` не возвращает ничего